SPECS/malaga/malaga.spec

```diff
@@ -1,13 +1,12 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-Name:           malaga
-Version:        7.12 
-Release:        31%{?dist}
 Summary:        A programming language for automatic language analysis
-
+Name:           malaga
+Version:        7.12
+Release:        32%{?dist}
 License:        GPLv2+
-URL:            http://home.arcor.de/bjoern-beutel/malaga/
-Source0:        http://home.arcor.de/bjoern-beutel/malaga/%{name}-%{version}.tgz
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://dynalabs.de/mxp/malaga/
+Source0:        https://cblmarinerstorage.blob.core.windows.net/sources/core/malaga-%{version}.tgz
```

Comment:
Why are we trying to use our blob storage now? It should be considered last resort (no external source available at all). We prefer external URLs as they tend to make updates a lot easier. Was there something wrong with ` https://dynalabs.de/mxp/assets/malaga-7.12.tgz`? It works without issues on my end.

Reply:
I found the failing cgmanifest check you're fighting with. Extending `grep` a bit should help. Try this in the check:

```bash
wget --spider --timeout=1 --tries=10 "${manifesturl}" 2>&1 | grep -qP "^(Remote file|File ‘.*’) exists(\.| and could contain.*)$"
```

Reply:
It works locally, but fails the pipeline tests every single time. Not sure why.

Reply:
> I found the failing cgmanifest check you're fighting with. Extending `grep` a bit should help. Try this in the check:
> 
> ```shell
> wget --spider --timeout=1 --tries=10 "${manifesturl}" 2>&1 | grep -qP "^(Remote file|File ‘.*’) exists.*"
> ```

Yeah, I tried messing around with it but was still running into issues; I didn't realize the slash was causing the issues. I'll add that change in :)